magefiles/utils.go

```diff
@@ -70,6 +61,7 @@ func sendHttpRequestAndParseResponse(url, method string, v interface{}) error {
 		return err
 	}
 	req.Header.Set("Authorization", fmt.Sprintf("token %s", os.Getenv("GITHUB_TOKEN")))
+	fmt.Println("DSADDDSDADASDDSASD")
```

Comment:
I'm guessing this is a temporary debugging line?

Reply:
damn, forgot to remove this line